### DISCUSSION TITLE
Add Depth Strider and Soul Speed Enchantments

### DISCUSSION THREAD

**Discussion by Omgise** [Roadhog360/Et-Futurum-Requiem#126] [Ideas]
Meow.

**Comment by GregoriusT** [comment]
Uh what is "South Speed"?

**Reply by Omgise** [reply]
> 呃，什么是“南速”？

It's a typo. It's soul speed（Because Google translation is used, there may be errors. Sorry）

**Comment by Roadhog360** [comment]
What did he mean by this

**Reply by Omgise** [reply]
> 他这是什么意思

Is to add two types of enchantments.😂（Because Google translation is used, there may be errors. Sorry）

**Comment by GregoriusT** [comment]
So I fixed the  Title. ^^

Now says Depth Strider and Soul Speed, instead of Depth Slider and South Speed. Google Translate is fun, isn't it.

**Reply by Omgise** [reply]
> 所以我修复了标题。^^
> 
> 现在说的是深度步进器和灵魂速度，而不是深度滑块和南方速度。谷歌翻译很有趣，不是吗。

Yes, it's interesting.  :D
